### DISCUSSION TITLE
Separator At Top Of Side-bar

### DISCUSSION THREAD

**Discussion by ChrisLeeThompson** [imfing/hextra#156] [Q&A] [answered]
I would like to add a separator at the top of the side-bar. Is this possible? I have attempted to edit the sidebar.html file to add an element at the top, but so far I am unsuccessful.

Does anyone know a simple way to do this?

Thank you for your assistance!

**Comment by imfing** [comment]
@12thankyou Hi, can you provide any example of what you were trying to achieve?

**Reply by ChrisLeeThompson** [reply]
My apologies for not providing more information earlier!

It might not be the best choice, but I am wanting to see what it would look like to have a separator/text at the top of the side-bar in a similar way as used in the Geekdocs Hugo theme.

Here is a screenshot from the Geekdocs theme. The "Navigation" separator/text is what I am looking to replicate with the Hextra theme.

![image](https://github.com/imfing/hextra/assets/47163544/fbae08b5-c808-4c7d-b029-3a945a4b52ff)

**Reply by imfing** [reply, marked as answer]
Thanks, I've created an issue to implement this new feature. https://github.com/imfing/hextra/issues/162

**Reply by ChrisLeeThompson** [reply]
Ok - thank you! I very much appreciate the work you have put into this theme - it is very good!

**Reply by imfing** [reply]
@12thankyou Would something like the following meet your requirements?

<img src="https://github.com/imfing/hextra/assets/5097752/767573f6-309e-4230-8720-02d4a55112f6" width="500">

**Reply by ChrisLeeThompson** [reply]
Yes, this is exactly what I was attempting to do! Thank you!
